src/modules/drawing/components/useModifyInteraction.composable.js

```diff
@@ -54,17 +56,78 @@ export default function useModifyInteraction(features) {
         wrapX: true,
     })
 
+    const continueDrawingInteraction = new DrawInteraction({
+        style: drawLineStyle,
+        type: 'LineString', // Only works for LineString
+        minPoints: 2,
+        stopClick: true,
+        // only left-click to draw (primaryAction)
+        condition: (e) => primaryAction(e),
+        wrapX: true,
+    })
+
+    watch(
+        () => store.state.drawing.editingMode,
```

Comment:
I'd prefer if you express your link to the store by binding it to a `computed` instead of using it directly